plugins/messagelayer/mana_plugin.go

```diff
@@ -759,40 +759,49 @@ func QueryAllowed() (allowed bool) {
 func loadSnapshot(snapshot *ledger.Snapshot) error {
 	consensusManaByNode := map[identity.ID]float64{}
 	accessManaByNode := map[identity.ID]float64{}
-	processOutputs := func(outputsWithMetadata []*ledger.OutputWithMetadata, areCreated bool) {
+	processOutputs := func(outputsWithMetadata []*ledger.OutputWithMetadata, baseVector map[identity.ID]float64, areCreated bool) {
 		for _, outputWithMetadata := range outputsWithMetadata {
 			devnetOutput := outputWithMetadata.Output().(devnetvm.Output)
-			var manaValue float64
-			devnetOutput.Balances().ForEach(func(color devnetvm.Color, balance uint64) bool {
-				if color != devnetvm.ColorIOTA {
-					return true
-				}
-				manaValue += float64(balance)
-				return true
-			})
+			balance, exists := devnetOutput.Balances().Get(devnetvm.ColorIOTA)
+			if !exists {
+				continue
+			}
 			outputMetadata := outputWithMetadata.OutputMetadata()
 			if areCreated {
-				consensusManaByNode[outputMetadata.ConsensusManaPledgeID()] += manaValue
-				accessManaByNode[outputMetadata.AccessManaPledgeID()] += manaValue
+				baseVector[outputMetadata.ConsensusManaPledgeID()] += float64(balance)
 			} else {
-				consensusManaByNode[outputMetadata.ConsensusManaPledgeID()] -= manaValue
-				accessManaByNode[outputMetadata.AccessManaPledgeID()] -= manaValue
+				baseVector[outputMetadata.ConsensusManaPledgeID()] -= float64(balance)
 			}
 		}
 
 		return
 	}
 
-	processOutputs(snapshot.OutputsWithMetadata, true /* areCreated */)
+	processOutputs(snapshot.OutputsWithMetadata, consensusManaByNode, true /* areCreated */)
```

Comment:
What are those comments for `/* areCreated */` , isn't this usually showed by the IDE?